ios/KMMIC/Sources/Presentation/Views/NetworkImage.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  NetworkImage.swift
+//  KMMIC
+//
+//  Created by MarkG on 21/04/2023.
+//  Copyright © 2023 Nimble. All rights reserved.
+//
+
+import SDWebImageSwiftUI
```

Comment:
Any reason why using SDWebImageSwiftUI? If you want to try something new, I prefer to use https://github.com/kean/Nuke

Reply:
@suho updated at [06755a2](https://github.com/markgravity/kmm-ic/pull/70/commits/06755a2920e55aafe9fd1096eb2827f3a21fcd21)